### ISSUE TITLE
Strapi not Work in Mac M1 chip

### ISSUE THREAD

**Issue by mehdiraized** [strapi/strapi#9121]
this command not work
```
yarn create strapi-app my-project --quickstart
```
error:
```
error /my-project/node_modules/sharp: Command failed.
Exit code: 127
Command: (node install/libvips && node install/dll-copy && prebuild-install) || (node-gyp rebuild && node install/dll-copy)
Arguments:
Directory: /my-project/node_modules/sharp
Output:
info sharp Detected globally-installed libvips v8.10.5
info sharp Building from source via node-gyp
/.nvm/versions/node/v14.15.4/lib/node_modules/npm/bin/node-gyp-bin/node-gyp: line 5: /usr/local/lib/node_modules/node-gyp/bin/node-gyp.js: No such file or directory
```

**Comment by petersg83** [comment]
The installation doesn't work on M1 chips for the moment. We are waiting for our dependencies to support it (https://github.com/lovell/sharp/issues/2460).

However, based on this PR https://github.com/strapi/strapi/pull/9106 there is a workaround:
"For users running on Apple Silicon M1, you may encounter errors thrown by sharp. You may need to re-install `libvps` or to build `sharp` manually following this [issue comment](https://github.com/lovell/sharp/issues/2460#issuecomment-751491241) in order to start the project."

**Comment by bassochette** [comment]
The issue can be bypassed by installing the libvips through brew
```
brew install libvips
```

**Comment by emma-r-slight** [comment]
I can verify that you can run strapi on an M1, I ran

`brew install vips`

and can now use Strapi on my machine

**Comment by 7alip** [comment]
I installed Homebrew both natively and under Rosetta 2 and added aliases in `~/.zshrc` file:
```bash
alias brew86="arch -x86_64 /usr/local/homebrew/bin/brew" // Rosetta 2 path
alias brewARM="/opt/homebrew/bin/brew" // Native path
```

If you have `vips` already installed, try to uninstall it from both if it exists:
```shell
brew86 uninstall vips
brewARM uninstall vips
```

Remove nvm/npm/yarn
```bash
rm -rf node_modules yarn.lock package-lock.json // Strapi project
rm -rf ~/.npm
rm -rf ~/.nvm
rm -rf ~/.config/yarn
```

Install `libvips`
```bash
brewARM install vips
```

Install nvm and node v14 again again
```bash
brewARM install nvm
nvm install 14
```

Make sure node process is arm64
```bash
node -p process.arch
arm64
```

**Comment by mehdiraized** [comment]
> I can verify that you can run strapi on an M1, I ran
> 
> `brew install vips`
> 
> and can now use Strapi on my machine

not work with nvm

**Comment by markkaylor** [comment]
I had an issue with nvm on M1 mac and was able to get it working.  Right click (ctrl click) your terminal application, select get info, and tick the "open using rosetta" option.  Restart terminal.  This worked for me using iTerm.  Hope it helps.

**Comment by mehdiraized** [comment]
https://github.com/lovell/sharp/issues/2604

**Comment by mehdiraized** [comment]
Please provide version 3.5.5 which solves this Sharp problem

**Comment by derrickmehaffy** [comment]
@alexandrebodin did we have a timeline for v3.5.5? https://github.com/strapi/strapi/milestone/145

**Comment by alexandrebodin** [comment]
Will release this week

**Comment by joelsamuelk** [comment]
@alexandrebodin any update on this?

**Comment by alexandrebodin** [comment]
Hi @joelsamuelk we release v3.6.0 this week with the fix. You can upgrade and confirm it works and I'll close thi issue

**Comment by joelsamuelk** [comment]
All seems to be working. Thanks @alexandrebodin

**Comment by derrickmehaffy** [comment]
Marking as closed as fixed in v3.6.0

**Comment by GraemeFulton** [comment]
here is a workaround i found for M1 mac if anyone still searching: https://forum.strapi.io/t/strapi-does-not-run-on-mac-m1/15563/3
